surveys/templates/partials/sidebar.html

```diff
@@ -46,6 +32,23 @@ <h3><b>Just Spaces</b></h3>
                     View collected data
                 </a>
             </li>
+            <li>
+                <a href="#createSurveySubmenu" data-toggle="collapse" aria-expanded="false" class="dropdown-toggle">
+                    <i class="fas fa-sm fa-home"></i>
+                    Admin
+                </a>
+                <ul class="collapse list-unstyled" id="createSurveySubmenu">
+                    <li>
+                        <a href="{% url 'agencies-create' %}">- Create Agency</a>
+                    </li>
+                    <li>
+                        <a href="{% url 'studies-create' %}">- Create Study</a>
+                    </li>
+                    <li>
+                        <a href="{% url 'locations-create' %}">- Create Location</a>
+                    </li>
+                </ul>
+            </li>
```

Comment:
I like this new menu a lot. Since our list views all have `Create new` buttons now, what do you think about having these links point to the list pages instead (such that the menu options would be `Agencies | Studies | Locations`)? My instinct is that the list view is more generally useful than the create view.

Reply:
I think that's a great idea. I'd been meaning to create basic List, Detail, and Delete views for Agency too, and I did that here so that this will work.